src/profile/stats/stats.styles.jsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'react-emotion';
+
+export const StatsContainer = styled('div')`
+  background-image: linear-gradient(-225deg, #a7a6cb 0%, #8989ba 52%, #8989ba 100%);
+  padding: 15px 20px;
+  color: white;
+  border-radius: 0.4em;
+`;
+
+export const Row = styled('div')`
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+
+  > div {
+    padding: 5px 10px;
+  }
+`;
+
+export const Divider = styled('hr')`
+  border-color: linear-gradient(120deg, #fdfbfb 0%, #ebedee 100%);
+`;
+
+export const ProgressName = styled('div')`
+  flex: 1;
+  text-align: right;
+  text-transform: uppercase;
+  font-family: 'Dosis', sans-serif;
+  font-weight: 700;
+`;
+
+export const ProgressBar = styled('div')`
+  font-family: 'Concert One', cursive;

```

Comment:
yeah def feel we should have some centralized file/folder for storing things like this

Reply:
👍